client/src/components/employer-form/BaselineList.js

```diff
@@ -0,0 +1,196 @@
+import React, { Fragment } from 'react';
+import CancelIcon from '@material-ui/icons/Cancel';
+import AddCircleIcon from '@material-ui/icons/AddCircle';
+import Typography from '@material-ui/core/Typography';
+import Accordion from '@material-ui/core/Accordion';
+import AccordionSummary from '@material-ui/core/AccordionSummary';
+import AccordionDetails from '@material-ui/core/AccordionDetails';
+import { InputFieldError, InputFieldLabel } from '../generic';
+import { Divider, Grid } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+import { useFormikContext } from 'formik';
+
+const useStyles = makeStyles((theme) => ({
+  dividerSpacing: {
+    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(2),
+  },
+  dividerTextSpacing: {
+    marginTop: theme.spacing(2),
+  },
+}));
+
+export const BaselineList = ({
+  field: { name, value, onChange, onBlur },
+  form,
+  label,
+  options,
+  ...props
+}) => {
+  const { setFieldValue } = useFormikContext();
+
+  const setValue = (option, result) => {
+    const mergedResult = { ...value, [option.value]: result }
+    setFieldValue(name, mergedResult);
+  };
+
+  const classes = useStyles();
+  const error = form.errors[name];
+
+  const convertValue = (value) => {
+    return typeof value === "undefined" ? '' : value
+  }
+
+  return (
+    <Fragment>
+      {options.map((option) => (
+        <Fragment key={option.value}>
+          <Accordion
+            expanded={value[option.value]?.add ? true : false}
+            onChange={() => {
+              setValue(option, {
+                ...value[option.value],
+                add: !value[option.value]?.add,
+              });
+            }}
+            {...props}
+          >
+            <AccordionSummary expandIcon={
+              !value[option.value]?.add ?
+                <AddCircleIcon color="primary" />
+                :
```

Comment:
interesting, is putting this on its own line a react convention? in other languages I've used it's usually inline so just for my future reference :)

Reply:
Depends on your line size, usually good lint doesn’t allow lines too big (100-200), so I'm used to not write lines too big :)